docs/core/install/linux-package-manager-opensuse15.md

```diff
@@ -36,6 +36,13 @@ sudo chown root:root /etc/zypp/repos.d/microsoft-prod.repo
 
 Update the products available for installation, then install the .NET Core SDK. In your terminal, run the following command.
 
+> [!IMPORTANT]
+> The .NET Core 3.1 package feed for openSUSE has a problem with the **krb5** dependency. Use the following command to install the correct dependencies.
```

Comment:
```suggestion
> The .NET Core 3.1 package feed for openSUSE has a problem with the **krb5** dependency. Use the following command to install the correct dependencies:
```

Reply:
@gewarren An editor recently corrected my `:` to introduce code blocks saying that they are for lists only. I did check the style guide and it only lists using `:` as a way to introduce a list. So because of the editors recommendation, I stopped using them for code blocks.